cmd/olm/main.go

```diff
@@ -257,10 +258,30 @@ func main() {
 		go monitor.Run(op.Done())
 	}
 
+	// Emit CSV metric
+	if err = ensureCSVMetric(logger, crClient); err != nil {
+		logger.WithError(err).Fatalf("error emitting CSV metric")
+	}
+
 	// Start the controller manager
 	if err := mgr.Start(ctx); err != nil {
 		logger.WithError(err).Fatal("controller manager stopped")
 	}
 
 	<-op.Done()
 }
+
+func ensureCSVMetric(logger *logrus.Logger, c *versioned.Clientset) error {
+	logger.Debug("Emitting CSV metric")
```

Comment:
`nit`: "Emitting metrics for existing CSVs" might be a more appropriate log message